cloud/openstack/os_networks_facts.py

```diff
@@ -33,10 +33,16 @@
     - "python >= 2.6"
     - "shade"
 options:
-   network:
+   name:
      description:
-        - Name or ID of the Network
```

Comment:
the underlying method that the module calls to get networks facts, is shade search_networks:

https://github.com/openstack-infra/shade/blob/master/shade/openstackcloud.py#L1136

That method accepts either a name or an ID, and return it.

So you should leave 'Name or ID of the Network'